### ISSUE TITLE
Quest: Accessibility

### ISSUE THREAD

**Issue by jonpitch** [b-lab-org/ember-cli-impact-core#122]
## Overview
The accessibility of our Ember application could be greatly improved with some attention. I think this is a good idea becase:

- It is aligned with B Lab's values
- a11y would increase the usability of our addon for a broader audience
- Increase team knowledge of accessibility for web development

## Tasks
- [ ] perform an audit
Some tools were introduced a while back to help us here. [ember-a11y](https://github.com/ember-a11y/ember-a11y) and [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) would be a good place to start. We should do a first pass to see what are our most common violations. Perhaps there's a common thing we do often that could be easily addressed. 

- [ ] placeholder for addressing violations
